### DISCUSSION TITLE
Version dockerisée

### DISCUSSION THREAD

**Discussion by PBMCHUGA** [vidboda/MobiDetails#35] [Ideas]
Bonjour,

La plateforme de biologie moléculaire du CHUGA a appris avec grand intérêt l’existence du MobiDetails.  Nous aimerions mettre en place un outil d’annotation et d’interprétation de variants dans le cadre du diagnostique routinier des maladies rares. 

L’idée est de pouvoir automatiser le feeding d’une base de données locale tout en permettant l’interprétation des variants. 

Nous pourrions contribuer à une version facilement déployable grâce à docker-compose si vous y voyez un intérêt. Nous contribuerions à l'instance principale et créant des ponts avec notre instance locale. 

Dans l’attente de vous lire,

Bien cordialement,

Clément B

PS : J’ai essayé de déployer une instance locale depuis le code source sans succès pour l’instant.

**Comment by vidboda** [comment]
Bonjour Clément,

merci de considérer MobiDetails avec intérêt. Il est en effet compliqué de déployer localement MobiDetails juste à partir du repo github. Il manque en effet la structure de la base de données.

Sur le principe, une dockerisation de Mobidetails serait très intéressante.

On peut discuter plus avant de vos besoins et voir comment procéder, je pense que ce sera mieux par email.


david
